tests/bft.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+scriptdir=$(dirname $(readlink -f $0))
+bf_dir=$(realpath "$scriptdir/../../boardfarm")
+bfp_dir=$scriptdir
+
+PYTHONPATH="$bfp_dir" python3 "$bf_dir/bft" "$@"
```

Comment:
You need to extend the PYTHONPATH if it exists already.

Also, better use exec.
```suggestion
if [ -n "$PYTHONPATH" ]; then
   PYTHONPATH="$bfp_dir:$PYTHONPATH"
else
   PYTHONPATH="$bfp_dir"
fi
export PYTHONPATH


exec python3 "$bf_dir/bft" "$@"
```